### ISSUE TITLE
Error Fetching Resource / Internal Server Error

### ISSUE THREAD

**Issue by songjuntaelikelion** [2016rshah/githubchart-api#32]
[2016rshah's Github chart](https://camo.githubusercontent.com/8023c46a01be9d86fcb1a312c7215048cac84f7f6c3d4a08124cd3031b509ab2/68747470733a2f2f676863686172742e72736861682e6f72672f323031367273686168)
The address above gave me the following error. 
    Error Fetching Resource

In my case I got the following error.
    Internal Server Error

I always appreciate your efforts.

**Comment by Jfaler** [comment]
I am also receiving the same error. Any update on this?

**Comment by 2016rshah** [comment]
I've tracked down the root of the issue, which has been raised here: https://github.com/akerl-archived/githubstats/issues/42

**Comment by 2016rshah** [comment]
Submitted a pull request to that repository, which will hopefully be the beginning of the solution: https://github.com/akerl-archived/githubstats/pull/43

**Comment by pluveto** [comment]
Is there any alternative app?

**Comment by 2016rshah** [comment]
Back up and running now :)

TL;DR: github changed the name of one of the fields in their HTML so the upstream scraping code needed to be updated. Huge thanks, as always, to @akerl <3
